aws_cloudtrail_rules/aws_key_compromised.py

```diff
@@ -0,0 +1,14 @@
+def rule(event):
+    # Capture PutUserPolicy
+    request = event.get('requestParameters')
+    return event.get('eventName').startswith(
+        'PutUserPolicy') and request.get('policyName').startswith('AWSExposedCredentialPolicy_DO_NOT_REMOVE')
+
+
+def dedup(event):
+    return event.get('userIdentity', {}).get('userName')
```

Comment:
```suggestion
    return event['userIdentity'].get('userName')
```

You can skip `.get()` for the required fields (bolded and [found here](https://docs.runpanther.io/supported-logs/aws#aws-cloudtrail))

Reply:
👍